### PR TITLE
Moved GetServiceLogFilePath into OrbitService main

### PR DESCRIPTION
**Pull request by antonrohr**
This is a small part of my effort to get rid of OrbitCore in OrbitService

### CODE REVIEW

File: OrbitCore/Path.cpp [hunk 1]
```diff
@@ -255,12 +255,6 @@ std::string Path::GetHome() {
   std::string home = GetEnvVar("HOME") + "/";
   return home;
 }
-
```
Comment (dimitry-) [inline OrbitCore/Path.cpp:258]: nit: maybe explain in the commit message that it is only being used on the service side anyways.

File: OrbitService/main.cpp [hunk 2]
```diff
@@ -38,7 +38,9 @@ void InstallSigintHandler() {
 }  // namespace
 
 int main(int argc, char** argv) {
-  const std::string log_file_path = Path::GetServiceLogFilePath();
+  std::filesystem::path var_log{"/var/log"};
+  std::filesystem::create_directory(var_log);
+  const std::string log_file_path = var_log / "OrbitService.log";
```
Comment (dimitry-) [inline OrbitService/main.cpp:43]: also maybe preserve a function here just make it static?

Comment (antonrohr) [reply OrbitService/main.cpp:43]: just to declutter the main method? Okay, will do

File: OrbitCore/Path.cpp (hunk 1 above)
Comment (antonrohr) [reply OrbitCore/Path.cpp:258]: will do on rebase

File: OrbitService/main.cpp (hunk 2 above)
Comment (antonrohr) [reply OrbitService/main.cpp:42]: done

**Pull request merged**